File: ThaiTuanERP2025.Server/ThaiTuanERP2025.Application/Finance/DTOs/CashOutCodeDto.cs
namespace ThaiTuanERP2025.Application.Finance.DTOs
{
	public record CashoutCodeDto(
		Guid Id, string Code, string Name, string? Description, bool IsActive,
		Guid CashoutGroupId, string CashoutGroupCode, string CashoutGroupName,
		Guid PostingLedgerAccountId, string PostingLedgerAccountNumber, string PostingLedgerAccountName
	);
}

File: ThaiTuanERP2025.Client/src/app/pages/finance/models/cash-out-code.dto.ts
export interface CashOutCodeDto {
      id: string;
      code: string;
      name: string;
      groupId: string; // FK --> CashOutGroup
      groupName?: string;
      isActive: boolean;
      description?: string;
}
export type CreateCashoutCodeRequest = Omit<CashOutCodeDto, 'id' | 'isActive' | 'groupName'>;
export type UpdateCashoutCodeRequest = Partial<Omit<CashOutCodeDto, 'id' | 'groupName'>>;